app/src/aiMembers/aiMembers.html

```diff
@@ -1,6 +1,112 @@
-<p>This is the partial for view 2.</p>
-<p>
-  Showing of 'interpolate' filter:
-  {{ 'Current version is v%VERSION%.' | interpolate }}
-</p>
-<div></div>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
I know this is for your local version, but just for the sake of argument. Only link in the original index.html.